### DISCUSSION TITLE
Are there any ideas to base Elementary in general on Debian Linux?

### DISCUSSION THREAD

**Discussion by RogerWeihrauch** [elementary/triage#340] [Q&A]
Hi to all

Since Elementary OS is a very, very fine one to have, and based on Ubuntu, which is also based on Debian, and, since Debian, actuall since Debian12, is very 'fast' on newest kernel and modules/Firmware, my Q is:

Will there be any interest, or timebase or aim to get Debian Linux as a basis for Elementary OS instead of Ubuntu?

I my eyes this has the following advantages:
1) Debian has a very huge Package repo itself and is also compatible with FlatPak, and others
2) Debian is very fast with new (kernel) modules and latest firmware
3) Debian is actually faster on integrating the (nearly) latest kernels.
(it runs on kernel 6.4.4 actually)

What do you think?

Regards,
Roger

**Comment by RMcNeely** [comment]
(I know that this has come up several times in the past so I'm trying to add my recollection from previous discussions).

IIRC the main advantage that Ubuntu has over Debian is the security team. For a long time the two have been very similar (though the whole snaps and flatpak debacle is making this less true today) so there wasn't a big need to go with Debian if there was added benefits of Canonical's work on security and the hardware enablement stack. 

Also at this point there's just a question of inertia. elementary has always been based on Ubuntu so there would need to be a very compelling reason to move. This is doubly true since the elementary team isn't huge so if there was a change it would probably mean dropping other work somewhere or delaying on releases.
